Nukeeper.BitBucketLocal/Models/AddReviewer.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace NuKeeper.BitBucketLocal.Models
+{
+    public class AddReviewer
```

Comment:
This class name seems a little strange would `PullRequestReviewer` be better?

The use of `List<AddReviewer>`  in the pull request model is why a change could be good

Reply:
I agree